v19.1/managed-build-a-python-app-with-kubernetes.md

```diff
@@ -0,0 +1,465 @@
+---
+title: Build a Python app with Kubernetes on Managed CockroachDB
+summary: Learn how to build a Python application with Managed CockroachDB and orchestrate it using Kubernetes.
+toc: true
+build_for: [managed]
+---
+
+This tutorial shows you how to run a [sample Python To-Do app](https://github.com/cockroachdb/examples-python/tree/master/flask-sqlalchemy) on a Managed CockroachDB cluster using the [Kubernetes](http://kubernetes.io/) orchestration system.
```

Comment:
would be good to say up front something like

"in this tutorial, your app will be running on a local kubernetes cluster via minikube. Similar steps can be used to deploy it on a non-local k8s cluster, such as GKE, EKS, RedHat, etc"

Reply:
To-do: Add Dockerfile and yaml file to GH repo